src/site/content/en/patterns/media/microphone-record/assets/body.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="icon"
+      href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>🎙️</text></svg>"
+    />
+    <title>How to record audio from a microphone</title>
```

Comment:
```suggestion
    <title>How to record audio from the user's microphone</title>
```

Let's make this symmetric to the screen recording title. Please change here and throughout.